components/esp_modem/include/cxx_include/esp_modem_dce_module.hpp

```diff
@@ -104,6 +104,15 @@ class GenericModule: public ModuleIf {
         pdp = std::move(new_pdp);
     }
 
+    /**
+     * @brief Simplified version of operator name (without the ACT, which is included in the command library)
+     */
+    command_result get_operator_name(std::string& name)
```

Comment:
```suggestion
    command_result get_operator_name(std::string const& name)
```

Reply:
This couldn't be made const ref I'm afraid, the name parameter gets updated in the function.